css/style.css

```diff
@@ -11,4 +11,92 @@ font-family: 'Raleway', sans-serif;
 
 *{
     font-family: 'Raleway', sans-serif;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
+.contact-us{
+    color:#487e95;
+    background-color:#f4f5db ;
+    width: 100%;
+}
+.contact-Container{
+    text-align: center;
+    margin: auto;
+    width: 60%;
+    padding:3%;
+    
+    
+}
+.contact-us-header{
+    font-weight:bold; 
+    color:#487e95 ;
+    margin: center;
+   padding: 3%;
+   font-size: 30px;
```

Comment:
pay attention to the code formatting, use prettier to format your code 

![Peek 2021-03-02 19-15](https://user-images.githubusercontent.com/47992412/109779657-d2071d00-7c0e-11eb-8870-b6bc3b6e4a92.gif)